docker/docker-compose.yml

```diff
@@ -121,6 +121,7 @@ services:
     expose:
       - "3000"
       - "9394"
+      - "12345"
```

Comment:
Not sure why this is here?

Reply:
Oh, thanks, that was an attempt a remote debugging.  Guess I missed it.